<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>channel1：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="channel1HorizontalSlider">
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="channel1SpinBox">
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>channel2：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="channel2HorizontalSlider">
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="channel2SpinBox">
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>channel3：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="channel3HorizontalSlider">
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="channel3SpinBox">
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>channel4：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="channel4HorizontalSlider">
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="channel4SpinBox">
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>channel5：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="channel5HorizontalSlider">
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="channel5SpinBox">
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>channel6：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="channel6HorizontalSlider">
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="channel6SpinBox">
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>channel7：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="channel7HorizontalSlider">
          <property name="maximum">
           <number>2000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="channel7SpinBox">
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="loseFlagCheckBox">
      <property name="text">
       <string>disconnected</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="dataFlagCheckBox">
      <property name="text">
       <string>data</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>channel1SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>channel1HorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>channel2SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>channel2HorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>channel3SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>channel3HorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>channel4SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>channel4HorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>channel5SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>channel5HorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>channel6SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>channel6HorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>channel7SpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>channel7HorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
